WDAC-Policy-Wizard/app/WDAC Wizard.csproj

```diff
@@ -202,13 +208,58 @@
     <Content Include="Resources\imgs\windows-logo.png" />
     <Content Include="Resources\imgs\wizard-ico.ico" />
     <Content Include="Resources\policyTemplates\AllowMicrosoft.xml" />
+    <None Update="priconfig.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
     <None Update="Properties\Settings.settings">
       <Generator>SettingsSingleFileGenerator</Generator>
       <LastGenOutput>Settings.Designer.cs</LastGenOutput>
     </None>
     <None Update="Resources\policyTemplates\cipolicy.xsd">
       <SubType>Designer</SubType>
     </None>
+    <None Update="RulesDict.xml">
```

Comment:
This is a really good suggestion. Instead of adding a second set of all of these xml files, could you update the csproj to copy the xml files from the existing directories?

For instance, 

```xml
<None Update="Resources\RulesDict.xml">
    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    <TargetPath>RulesDict.xml</TargetPath>
  </None>
```